fast64_internal/sm64/sm64_constants.py

```diff
@@ -2005,3 +2005,19 @@ def __init__(self, geoAddr, level, switchDict):
     (5737600, "207 - Forwards spinning flip"),
     (5740584, "208 - Triple jump fly"),
 ]
+
+sm64_world_defaults = {
+    "geometryMode": {
+        "zBuffer": True,
+        "shade": True,
+        "cullBack": True,
+        "lighting": True,
+        "shadeSmooth": True,
+    },
```

Comment:
I don't have a lot of knowledge but I think you should also add this to the oot world defaults since it defaults to false for all of these

Reply:
Already happening, when not set the from dict functions use the ucode defaults

Reply:
Wait @Yanis42 I was going to merge just now but by this you meant oot defaults them to false right? Wanted to double check

Reply:
sorry for the late answer I forgot TWICE in a row to reply lol stoopid brain

the oot defaults for ``geometryMode`` were the same as sm64 afaict so as long as it's stays on ``True`` it should be fine